db/dataGenerator.js

```diff
@@ -6,10 +6,6 @@ var randomProduct = faker.commerce.product;
 var imgBase = 'https://picsum.photos/200/300/?'
 var randomNum = faker.random.number;
 
-var stream = fs.createWriteStream('db/products.csv', { flags: 'a' });
-stream.write(`"productName", "bannerImageUrl", "productImageUrls"
-`);
-
 prodImgUrlsNum = () => {
```

Comment:
is this function needed?

Reply:
I can do it in-line within the singleData function, thanks for catching this!